keycloak/sms-provider/src/test/java/org/sunbird/keycloak/rest/RequiredActionLinkProviderTest.java

```diff
@@ -74,7 +74,7 @@ public static void setUp() throws Exception {
   @Test
   public void checkRealmAdminAccessForUnAuthorized() throws Exception {
     PowerMockito.whenNew(AppAuthManager.class).withAnyArguments().thenReturn(authMangr);
-    PowerMockito.when(authMangr.authenticateBearerToken(session, model)).thenReturn(null);
+    PowerMockito.when(authMangr.authenticateIdentityCookie(session, model)).thenReturn(null);
```

Comment:
Same comment given in earlier code. What is the impact with this change?

Replacing authenticateBearerToken with authenticateIdentityCookie

Reply:
Now this is reverted back with proper method changes, done in the latest commit.